### PR TITLE
Add wildcard option for modelName in dpta #563418

### PR DESCRIPTION
**Pull request by hnn0003**

### CODE REVIEW

**Review by MarcNeely — APPROVED** [review]

File: test-apps/display-performance-test-app/README.md [hunk 1]
```diff
@@ -56,6 +56,8 @@ When you chose to produce images through an 'image' or 'both' test, the program
...
 
+If you wish to specify multiple models in the same folder, you can use a asterisk wildcard instead of a specific model name to test multiple models matching the given wildcard. This is NOT case-sensitive. For example, to test all models, set modelName to "*". To test all models with the word 'Edge' in them, set modelName to "*edge*". To test models starting with the word edge, use "edge*". If no modelName is specified, it will default to testing all models in the given iModelLocation. This currently does NOT work when using an iModelHub location with iModelHubProject instead of a local directory with iModelLocation.
```
Comment (pmconne) [inline test-apps/display-performance-test-app/README.md:59]: By "model" you mean "iModel", correct?

File: test-apps/display-performance-test-app/src/frontend/DisplayPerformanceTestApp.ts [hunk 2]
```diff
@@ -1041,6 +1041,24 @@ async function signIn(): Promise<boolean> {
   return retPromise;
 }
 
+async function getAllMatchingModels(testConfig: DefaultConfigs): Promise<string[]> {
+  if (!testConfig.iModelLocation || !testConfig.iModelName?.includes("*"))
+    return [testConfig.iModelName!];
+
+  const matchingFilesJson = await DisplayPerfRpcInterface.getClient().getMatchingFiles(testConfig.iModelLocation, testConfig.iModelName ?? "*");
+  const matchingFiles = JSON.parse(matchingFilesJson);
+  const matchingModels: string[] = [];
```
Comment (pmconne) [inline test-apps/display-performance-test-app/src/frontend/DisplayPerformanceTestApp.ts:1050]: These are files (iModels), not models.

**Review by pmconne — CHANGES_REQUESTED** [review]
An iModel can contain any number of models. Please use "iModel" to refer to the file and "model" to refer to individual models within it.

**Review by pmconne — APPROVED** [review]
I still see a bunch of "model" in the code where you mean "iModel" but ok.

**Pull request merged**